### PR TITLE
Validate chart is updating

### PR DESCRIPTION
**Pull request by meeyun-deriv**
This PR aims to validate that the chart is updating and not broken. The process involves the following steps:

1. Open the Deriv app and create the first screenshot.
2. Wait for 2 seconds (almost all symbols will have new ticks within 2 seconds).
3. Create the second screenshot.
4. Compare both screenshots; they should have some detected pixel differences.

Additional information:

1. The screenshots exclude expected live update elements, such as contract price and timestamp in the footer.
2. If the chart is not loaded at all or the feed is stopped, both screenshots will be identical. Thus, the mismatched pixel count will be 0 (this has been tested in QA10 by stopping the feed).

When changes detected, test should pass:
image
image

When there is no changes, test should fail:
image

ClickUp card: https://app.clickup.com/t/20696747/DTRA-923

Test outcome from github action:
image

### CODE REVIEW

File: cypress/e2e/wip/dtrader/chartStreaming.cy.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+describe('QATEST-6247 - Verify if chart streaming is working with feeds', () => {
+    beforeEach(() => {
+        cy.c_visitResponsive("", "large")
+        localStorage.setItem("config.server_url", Cypress.env("configServer"))
+        localStorage.setItem("config.app_id", Cypress.env("configAppId"))
+    })
+
+    it('should update the line chart', () => {
+        cy.c_visitResponsive
```
Comment (markodonnell-deriv) [inline cypress/e2e/wip/dtrader/chartStreaming.cy.js:13]: This line should error, and probably was a mistake as it's in the befreEach?

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
Some minor comments

File: cypress/e2e/wip/dtrader/chartStreaming.cy.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import '@testing-library/cypress/add-commands'
+import '../../../support/dtrader'
+
+
+describe('QATEST-6247 - Verify if chart streaming is working with feeds', () => {
```
[the remark this answers is not in the document]
Comment (meeyun-deriv) [reply cypress/e2e/wip/dtrader/chartStreaming.cy.js:5]: here is the click up task https://app.clickup.com/t/20696747/QATEST-6247

File: cypress/e2e/wip/dtrader/chartStreaming.cy.js (hunk 1 above)
Comment (meeyun-deriv) [reply cypress/e2e/wip/dtrader/chartStreaming.cy.js]: thanks @markodonnell-deriv , you are right, it shouldn't be there. I have removed it.

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Pull request merged**